crates/storage/errors/src/provider.rs

```diff
@@ -172,14 +172,13 @@ impl From<UnifiedStorageWriterError> for ProviderError {
     }
 }
 
-#[cfg(feature = "std")]
-impl std::error::Error for ProviderError {
-    fn source(&self) -> Option<&(dyn std::error::Error + 'static)> {
+impl core::error::Error for ProviderError {
```

Comment:
is the std feature here (and in the other changed files) used for anything in their respective crates anymore?

Reply:
Yes normally I checked and I see in the two or three crates concerned another use either for display, `std::path::PathBuf` or another use